app/models/application_record.rb

```diff
@@ -50,6 +50,34 @@ def self.with_lax_group_rules
       connection.execute 'SET SESSION sql_mode = @old_sql_mode'
     end
   end
+
+  def self.useful_err_msg
+    [
+      'The inverted database guide has found an insurmountable problem. Please poke it with a ' \
+        'paperclip before anyone finds out.',
+      'The modular cable meter has found a problem. You need to kick your IT technician in the ' \
+        'shins immediately.',
+      'The integral output port has found a problem. Please take it back to the shop and take ' \
+        'the rest of the day off.',
+      'The integral expansion converter has encountered a terminal error. You must take legal ' \
+        'advice urgently.',
+      'Congratulations. You have reached the end of the internet.',
+      'The Spanish Inquisition raised an unexpected error. Cannot continue without comfy-chair-interrogation.',
+      'The server halted in an after-you loop.',
+      'A five-level precedence operation shifted too long and cannot be recovered without data loss. ' \
+        'Please re-enable the encryption protocol.',
+      'The server\'s headache has not improved in the last 24 hours. It needs to be rebooted.',
+      'The primary LIFO data recipient is currently on a holiday and will not be back before next Thursday.',
+      'The operator is currently trying to solve their Rubics cube. We will come back to you, when the ' \
+        'second layer is completed.',
+      'The encryption protocol offered by the client predates the invention of irregular logarithmic ' \
+        'functions.',
+      'The data in the secondary (backup) user registry is corrupted and needs to be re-filled with ' \
+        'random data again.',
+      'This community has reached a critical mass and collapsed into a black hole. Currently trying to ' \
+        'recover using Hawking radiation.'
```

Comment:
And the one I've recently added for pattern-matching: "Operations are on pause while we attempt to recapture the codidactyl. Please hold."